### PR TITLE
Stress tensor

### PR DESCRIPTION
**Pull request by Stefaanhess**

### CODE REVIEW

**Comment by Stefaanhess** [comment]
I also updated the docs, in order to explain the new loss-tradeoff for custom datasets. The new tradeoff is now defined with:

--rho property=1. derivative=0.5 stress=0.5 ...

**Comment by ktschuett** [comment]
@stefaanhess could you add an alias for negative derivatives named force? Because for derivative it is not clearly defined wrt what.

**Comment by Stefaanhess** [comment]
I added a force alias. Instead of setting --derivative forces --negative_dr one can now use --forces forces.
Also one can use forces when defining the tradeoff:
--rho property=0.9 derivative=0.1 is the same as --rho property=0.9 force=0.1

Does this help?

File: src/schnetpack/__init__.py [hunk 1]
```diff
@@ -66,6 +68,7 @@ class Properties:
     required_grad = {
         energy: [],
         forces: [position],
+        stress: [cell],
```
Comment (ktschuett) [inline src/schnetpack/__init__.py]: Should this be 'displacement'?

**Comment by mgastegger** [comment]
Fixed a few things.

**Pull request merged**